common/app/views/main.scala.html

```diff
@@ -43,7 +45,7 @@
             @fragments.header(metaData)
         </div>
 
-        @if(showAdverts && adBelowNav) {
```

Comment:
I changed the logic so that `adBelowNav` can only be true if `showAdverts` is true.